### DISCUSSION TITLE
Simple way to identify changes in the REST API

### DISCUSSION THREAD

**Discussion by codaamok** [shlinkio/discussions#104] [Help wanted] [answered]
Beyond reviewing the release notes for each release of Shlink, is there an easy way to identify what changes are made in its REST API? I vibe the swagger 'manifest' might help me here, but I know zero about swagger to know if there's worthwhile chasing as an idea or not.

**Comment by codaamok** [comment, marked as answer]
In the end, I ended up comparing the .json files of the swagger specification. Pretty painful. However, I did see this Java tool which looks cool: https://github.com/Sayi/swagger-diff

**Comment by acelaya** [comment]
Yeah, right now the only option is following the changelog, but I admit it's not ideal if you need to compare non-consecutive versions.
